src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -820,6 +978,39 @@ private static ImmutableArray<RefKind> GetRefKindsOrNull(ArrayBuilder<RefKind> r
             }
         }
 
+        private static readonly DiagnosticBag s_notUsedDiagnosticBagForDerivingArgumentsForIOperation = new DiagnosticBag();
+
+        private static void AppendMissingOptionalArguments(SyntaxNode syntax,
+            ImmutableArray<ParameterSymbol> missingParameters,
+            Binder binder,
+            ArrayBuilder<BoundExpression> argumentsBuilder,
+            ArrayBuilder<ArgumentKind> argumentKindsBuilder,
+            ArrayBuilder<ParameterSymbol> parametersBuilder,
+            ThreeState enableCallerInfo = ThreeState.Unknown)
```

Comment:
>hreeState enableCallerInfo = ThreeState.Unknown [](start = 13, length = 47)

It looks like this parameter is never passed explicitly. Are you certain that Unknown value gives the result that we want? If so, please remove the parameter.